jsk_spot_robot/README.md

```diff
@@ -27,51 +27,39 @@ cd $HOME/catkin_ws/src
 wstool init .
 wstool merge -t . https://github.com/sktometometo/jsk_robot/raw/develop/spot/jsk_spot_robot/jsk_spot.rosinstall
 wstool update
-rosdep update
-rosdep install -y -r --from-paths . --ignore-src
-pip3 install -r jsk-ros-pkg/jsk_robot/jsk_spot_robot/requirements.txt
 cd $HOME/catkin_ws
```

Comment:
I think many students use `~/catkin_ws` as default ros workspace, which means built as python2. So I think it would be better to set another name like `spot_ws`, `mkdir -p ~/spot_ws/src` .